packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -140,17 +140,23 @@ const Toolbar = React.memo(
             selectedNode={selectedNode}
             onFocusOnSubtree={onFocusOnSubtree}
           />
-          <Divider />
-          {!disableChangingDisplay && (
-            <ViewSection
-              flamegraphType={flamegraphType}
-              showMode={showMode}
-              view={view}
-              updateView={updateView}
-            />
-          )}
-          <Divider />
-          {ExportData}
+          {!disableChangingDisplay ? (
```

Comment:
Nit, but I recommend writing these boolean flags in the positive form.

Eg `enableChangingDisplay`

I found it's easier to read the different cases
`enableChangingDisplay` -> enable
`!enableCgangingDisplay` -> disable

Now compared to 
`disableChangingDisplay` -> is changing display enabled? no
`!disableChangingDisplay` -> double negative, we are disabling disabling changing display, which here means we are Enabling changing display

Reply:
updated